### ISSUE TITLE
Polling Rate resetting ROG Gladius II

### ISSUE THREAD

**Issue by NIKHIL-WR34** [seerge/g-helper#1927]
### Rules

- [X] I made myself familiar with the <a href='https://github.com/seerge/g-helper?tab=readme-ov-file'>Readme</a>, <a href='https://github.com/seerge/g-helper/wiki/FAQ'>FAQ</a> and <a href='https://github.com/seerge/g-helper/wiki/Troubleshooting'>Troubleshooting</a>.
- [X] I understand that, if insufficient information was provided, my issue will be closed without an answer.

### What's wrong?

Whenever I try to change the polling rate of my ROG Gladius II from 250hz to 1000hz it simply does not save it. When you open the peripheral tab again it shows 250hz.  

### How to reproduce the bug?

1) Click On Peripherals
2) Change Polling Rate from 250hz to 1000hz
3) Close the window
4) Open the Peripherals tab again
5) Polling Rate will be again reset to 250hz

### Logs

[log.txt](https://github.com/seerge/g-helper/files/13941072/log.txt)

### Device and Model

Asus ROG Scar 15 2021 G533QS

### Additional information.

![image](https://github.com/seerge/g-helper/assets/69037171/9e9e1bad-490c-453c-ae20-fe12c6cb38f6)

![image](https://github.com/seerge/g-helper/assets/69037171/b7036095-4663-4593-8d1e-7e3215a37f89)

### Armoury Crate

Installed

### Asus Services

6

### Version

0.146.0

### OS

Windows 10 22H2

**Comment by NIKHIL-WR34** [comment]
@seerge I also want to point out that even after updating this certain driver highlighted in the screenshot below it keeps on showing 1 update is available. Any way to fix this issue?
![image](https://github.com/seerge/g-helper/assets/69037171/04d1e40b-8bc8-440c-bf45-3e70095828d7)

**Comment by seerge** [comment]
@NIKHIL-WR34 that's not related anyhow to the mouse. App just compares driver version between what ASUS website says it is and what is in the system.

It could be that Asus just has incorrect version listed on website (while actual driver download has different one, for example with different order of digits, or dots between them) :) So that's "normal" ))

**Comment by NIKHIL-WR34** [comment]
@IceStormNG .. yes even after clicking on sync with the mouse it resets to 250hz. It's already set to 1000hz in AC, technically it should apply those settings when I click on sync right? Also, I would like to mention when Angle snapping is enabled in G Helper as well cursor seems like it's stuck and moves inch by inch at a certain moment. As mentioned Angle snapping is already in AC as well. So it's better not to enable Angle Snapping in GHelper or Its fixable?

**Comment by IceStormNG** [comment]
Great that it works.

It's not really a pre-release. I just fixed it with something I learned from another mouse a few days ago. If you say that all features on this mouse work fine in GHelper, I will push it and the next update of GHelper will contain this fix.
The version I work on (which I uploaded for testing) is often a bit behind because I'm too lazy to pull from upstream for testing purpose.
